frontend/src/component/application/ApplicationChart.tsx

```diff
@@ -163,6 +181,18 @@ export const ApplicationChart = ({ data }: IApplicationChartProps) => {
                             >
                                 {applicationName}
                             </Typography>
+                            <StyledIconRow>
```

Comment:
would it be more readable if we listened to codescene and avoided large function/large JSX which is also functions?